File: JS_OOPS/class.js
class Student{
    constructor(name , age , education){
        this.name =name;
        this.age =age ;
        this.education = education;

    }
    display(){
        console.log(`student name :${this.name}`);
        console.log(`student age : ${this.age}`);
        console.log(`student education : ${this.education}`);
    }
}
let  s1 = new  Student ("rohit",22,"MCA");
s1.display();

File: CPP_OOPS/class.cpp
#include <iostream>
#include <string>
using namespace std;

class Student{
    //properties
     string  name;
     string education;
     int age;
     double mark;

     public:
     //methods , member functions..
     Student(string n, string edu , int a , double m){
        this->name = n;
        this->education = edu;
        this->age = a;
        this->mark = m;

     }

    void display(){
       cout<<"student name:"<<name<<endl;
       cout<<"student qualification :"<<education<<endl;
       cout<<"student age:"<<age<<endl;
       cout<<"student marks:"<<mark<<endl;
     }
};
int main(){
    Student s1("rohit","MCA",22,66.90);
    s1.display();
    return 0;
}